### ISSUE TITLE
Example on Mac can't start

### ISSUE THREAD

**Issue by xudesheng** [tauri-apps/tauri#825]
**Describe the bug**
https://github.com/tauri-apps/tauri/tree/dev/tauri/examples/communication
when I use `npx tauri dev` to start this demo, it fails with error message:
`
info sharp Using cached /Users/xxxx/.npm/_libvips/libvips-8.9.1-darwin-x64.tar.gz
  ✔ optipng pre-build test passed successfully
  ✔ pngquant pre-build test passed successfully
  ✔ zopflipng pre-build test passed successfully
[tauri]: running dev
(node:8740) UnhandledPromiseRejectionWarning: /Users/xxxx/.npm/_npx/8740/lib/node_modules/tauri/node_modules/fs-extra/lib/mkdirs/make-dir.js:85
      } catch {
              ^

SyntaxError: Unexpected token {
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:617:28)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/xxxx/.npm/_npx/8740/lib/node_modules/tauri/node_modules/fs-extra/lib/mkdirs/index.js:3:44)
(node:8740) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:8740) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
`

**To Reproduce**
1. git clone --depth=1
2. cd tauri/examples/commuication
3. npx tauri dev

**Expected behavior**
demo app window should pop up.

**Screenshots**

**Platform and Versions (please complete the following information):**
Operating System - Darwin(19.5.0) - darwin/x64

Node.js environment
  Node.js - 8.12.0
  tauri.js - 0.9.1

Rust environment
  rustc - 1.41.0
  cargo - 1.41.0
  tauri-bundler - 0.8.5

Global packages
  NPM - 6.14.4
  yarn - Not installed

App directory structure
/dist
/src-tauri

App
  tauri.rs - path:../../.. [0.7.5] (no lockfile)
  mode - no-server
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: http: https: 'unsafe-eval' 'unsafe-inline'
  Windows - Edge
  distDir - ../dist
  devPath - ../dist

**Comment by nothingismagick** [comment]
I saw REALLY strange behavior with npx today. Can you please install normally with yarn and ru. `yarn tauri dev` and report back?

**Comment by xudesheng** [comment]
> I saw REALLY strange behavior with npx today. Can you please install normally with yarn and ru. `yarn tauri dev` and report back?

thank you, `yarn tauri dev` works as expected.

**Comment by lucasfernog** [comment]
I guess we need to remove "npx" from our docs lol

**Comment by nothingismagick** [comment]
Yeah, I was against it then, and now it's totally clear. @Laegel - can you please purge every. Mention of npx from our docs?

**Comment by xudesheng** [comment]
> I guess we need to remove "npx" from our docs lol

I just started to learn ReactJS from yesterday and Tauri today. I added Tauri to the ReactJS tutorial and `npx tauri dev` works as expected but I'm not sure why `npx` doesn't work for Tauri demo.

I wish Tauri can provide an end-to-end tutorial like ReactJS does to beginner like me. :)


![image](https://user-images.githubusercontent.com/8742037/87355215-fdc57d80-c52d-11ea-8c1d-acd18c15a922.png)
